### PR TITLE
Add Simplified Chinese

### PR DESCRIPTION
**Pull request by hjthjthjt**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<resources>
+    <string name="app_name">Tap, Tap</string>
+    <string name="xposed_desc">哥伦布后门</string>
+    <string name="accessibility_service_description">TapTap 无障碍五福\n\n<b>重要提示：应用不会收集您的数据，此服务仅用于识别手势活动并提供与设备的互动支持。</b></string>
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml:4]: 应该是无障碍服务吧？

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    <string name="action_launch_assistant_desc">启动您默认的语音助手</string>
+    <string name="add_action_category_info">高级操作，如 Tasker 集成，即将推出!</string>
+
+    <string name="accessibility_info_off">无障碍服务被禁用</string>
+    <string name="accessibility_info_off_desc">手势将无法工作。点击以启用</string>
+    <string name="accessibility_info_on">无障碍服务已启用</string>
+    <string name="accessibility_info_on_desc">如果您想禁用手势，点击以禁用服务</string>
+    <string name="accessibility_info_toast">在无障碍服务列表中选择「Tap, Tap」启用它，然后返回应用以继续</string>
+    <string name="menu_alpha">此应用处于 Alpha 阶段。点击了解详情。</string>
+    <string name="bs_alpha">这款应用程序还处于早期开发阶段，目前已经发布为了 Alpha 版本，可在众多设备上进行测试。如果您遇到了 bug，请在 GitHub 上提交一个 issue。如果您希望能有一些新功能，可以在 XDA 上的帖子里告诉我，或者同样在 GitHub 上提交。\n\n请在 XDA 上查看，无论是在实验室还是在这个帖子里，一旦有更新，就会立即更新。\n\n感谢您在开发过程中的耐心等候。</string>
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml:94]: “发布为了 Alpha 版本”语句不通顺

File: app/src/main/res/values-zh-rCN/strings.xml (hunk 1 above)
Comment (hjthjthjt) [reply app/src/main/res/values-zh-rCN/strings.xml:4]: 确实……没检查，怪我怪我……

**Comment by KieronQuinn** [comment]
Hi, I added a load of new actions and gates in 0.2 alpha so rather than merge in a half-complete translation, please update the strings to include the new text. 

Thanks for your help!

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    <string name="setting_gates_desc">禁止手势在某些情况下被识别到并激活动作</string>
+    <string name="setting_feedback">反馈</string>
+    <string name="setting_feedback_desc">定制设备在识别到手势时通知您的方式</string>
+    <string name="setting_gesture_model">设备模型</string>
+    <string name="setting_gesture_model_desc">选择机器学习组件使用的设备模型</string>
+    <string name="setting_feedback_vibrate">振动</string>
+    <string name="setting_feedback_vibrate_desc">当识别到手势并执行动作时，振动一次</string>
+    <string name="setting_feedback_wake">唤醒设备</string>
+    <string name="setting_feedback_wake_desc">当屏幕关闭时唤醒设备（注意：这会干扰「锁屏」动作！）</string>
+
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml:21]:
```
    <string-array name="device_model_keys">
        <item>Pixel 3 XL</item>
        <item>Pixel 4</item>
        <item>Pixel 4 XL</item>
    </string-array>

```

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    <!-- Gates -->
+    <string name="gate_charging_state">充电时</string>
+    <string name="gate_charging_state_desc">当设备刚处于充电状态时，暂时禁用手势半秒钟</string>
+    <string name="gate_telephony_activity">通话时</string>
+    <string name="gate_telephony_activity_desc">当设备在通话时，禁用手势。仅支持标准电话，不支持网络电话，例如：WhatsApp</string>
+    <string name="gate_usb_state">USB device</string>
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml]: USB 设备

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    <string name="bs_alpha_title">Alpha</string>
+
+    <string name="category_about">关于</string>
+    <string name="about">%1s v%2s</string>
+    <string name="about_summary">由 Kieron Quinn / Quinny899 @ XDA 开发</string>
+    <string name="xda_thread">XDA 帖子</string>
+    <string name="github">GitHub</string>
+    <string name="libraries">第三方库</string>
+    <string name="donate">捐赠</string>
+    <string name="twitter">在 Twitter 关注我</string>
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml:107]:
```
  <string name="tasker_config_label">双击背面</string>
    <string name="action_tasker_task">Tasker 任务</string>
    <string name="action_tasker_task_desc">运行指定的 Tasker 任务</string>
    <string name="action_tasker_event_toast">Tasker 未安装，或安装的版本不支持</string>
    <string name="bs_tasker_content">为了在不使用「事件」插件的情况下启动 Tasker 任务，您必须在 Tasker 设置中开启「允许外部访问」选项。如果您未开启此选项，那么本动作可能会被始终跳过。</string>
    <string name="bs_tasker_title">需要权限</string>
    <string name="bs_tasker_negative">稍后</string>
    <string name="bs_tasker_positive">打开 Tasker 设置</string>
    <string name="picker_launch_camera">启动相机</string>
    <string name="action_volume_panel">显示音量面板</string>
    <string name="action_volume_panel_desc">显示系统音量面板，并显示音频调整选项</string>
    <string name="action_volume_up">增大音量</string>
    <string name="action_volume_up_desc">增大当前音频流的音量</string>
    <string name="action_volume_down">减小音量</string>
    <string name="action_volume_down_desc">减小当前音频流的音量</string>
    <string name="action_volume_toggle_mute">静音开关</string>
    <string name="action_volume_toggle_mute_desc">启动 / 关闭当前音频流的静音</string>
    <string name="action_wake_device">唤醒设备</string>
    <string name="action_wake_device_desc">当屏幕关闭时，唤醒设备。这将覆盖「反馈」中的设置；如果屏幕已经开启，将跳过该动作；「屏幕关闭时」门槛开启时，该动作将失效。</string>
```

**Comment by hjthjthjt** [comment]
> Hi, I added a load of new actions and gates in 0.2 alpha so rather than merge in a half-complete translation, please update the strings to include the new text.
> 
> Thanks for your help!

I finished.🌚👌

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    <string name="gate_charging_state">充电时</string>
+    <string name="gate_charging_state_desc">当设备刚处于充电状态时，暂时禁用手势半秒钟</string>
+    <string name="gate_telephony_activity">通话时</string>
+    <string name="gate_telephony_activity_desc">当设备在通话时，禁用手势。仅支持标准电话，不支持网络电话，例如：WhatsApp</string>
+    <string name="gate_usb_state">USB device</string>
+    <string name="gate_usb_state_desc">当有 USB 设备插入时，禁用手势半秒钟。例如：U盘插入</string>
+    <string name="gate_camera_visibility">相机启动时</string>
+    <string name="gate_camera_visibility_desc">当相机开启时，暂时禁用手势。注意：这将在执行动作时跳过「启动相机」动作</string>
+    <string name="gate_power_state">屏幕关闭时</string>
+    <string name="gate_power_state_desc">当屏幕处于关闭状态时，暂时禁用手势</string>
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml:32]:
```
<string name="gate_power_state_inverse">屏幕打开时</string>
<string name="gate_power_state_inverse_desc">当屏幕处于开启状态时，暂时禁用手势</string>
```

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+    <string name="action_recents">打开最近任务</string>
+    <string name="action_recents_desc">通过模拟按下最近任务键来打开最近任务。在支持此功能的设备上，若最近任务已经处于打开状态，则使用此动作会切换至第一个后台应用。</string>
+    <string name="action_notifications">展开通知</string>
+    <string name="action_notifications_desc">展开通知面板</string>
+    <string name="action_quick_settings">快速设置</string>
+    <string name="action_quick_settings_desc">展开快速设置面板</string>
+    <string name="action_screenshot">截图</string>
+    <string name="action_screenshot_desc">使用系统界面截图</string>
+    <string name="action_lock_screen">锁屏</string>
+    <string name="action_lock_screen_desc">通过模拟按下电源键将设备锁屏</string>
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml:53]:
```
    <string name="action_toggle_pause">播放 / 暂停</string>
    <string name="action_toggle_pause_desc">播放 / 暂停当前正在播放的音频</string>
    <string name="action_previous">上一首</string>
    <string name="action_previous_desc">播放上一首音频</string>
    <string name="action_next">下一首</string>
    <string name="action_next_desc">播放下一首音频</string>
```

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+    <string name="setting_gesture_sensitivity">灵敏度</string>
+    <string name="setting_gesture_sensitivity_desc">此应用所使用的 Google 代码目前没有实现灵敏度功能。我正在积极地研究如何实现它，在未来版本中它也许将可用。</string>
+    <string name="bs_help_action">当您敲击设备背面两次时运行的任务，即为「动作」。动作将按照列表顺序从上往下执行。如果某个动作成功运行了，则其他的动作将不会运行。然而，若是一个动作无法运行（例如：在已经启动了语音助手的情况下，想运行「启动语音助手」）时，将会运行它下面的那个任务，以此类推，直到其中一个动作成功运行为止。\n\n在该应用程序的未来版本中，您将能够向动作添加门槛(请参阅此应用的「门槛」菜单)，以允许对某些动作何时运行进行更为灵活的控制。</string>
+    <string name="bs_help_action_title">什么是动作？</string>
+    <string name="bs_help_gate">门槛可以阻止手势运行任何动作。为了运行任何动作，您必须打开所有已启用的门槛。\n如果当前满足某个场景，例如：屏幕处于关闭状态，而「屏幕关闭时」处于启用状态时，则不会运行任何动作。您可以添加任意数量的门槛，并使用开关来启用或者禁用它们（没有必要删掉它们再重新添加），以节省时间并提高效率。</string>
+    <string name="bs_help_gate_title">什么是门槛？</string>
+    <string name="help">帮助</string>
+    <string name="action_launch_assistant">启动语音助手</string>
+    <string name="action_launch_assistant_desc">启动您默认的语音助手</string>
+    <string name="add_action_category_info">高级操作，如 Tasker 集成，即将推出!</string>
```
Comment (Asstiff) [inline app/src/main/res/values-zh-rCN/strings.xml]: 删除第 80 行，改为
```
<string name="action_tasker_event">Tasker 事件</string>
<string name="action_tasker_event_desc">触发 Tasker 事件插件</string>
```

**Comment by KieronQuinn** [comment]
Thanks for your help 👍

**Pull request merged**